typescript/sdk/src/deploy/verify/PostDeploymentContractVerifier.ts

```diff
@@ -0,0 +1,50 @@
+import { debug } from 'debug';
+
+import { ExplorerFamily } from '../../metadata/chainMetadataTypes';
+import { MultiProvider } from '../../providers/MultiProvider';
+import { ChainMap } from '../../types';
+import { MultiGeneric } from '../../utils/MultiGeneric';
+
+import { ContractVerifier } from './ContractVerifier';
+import { BuildArtifact, CompilerOptions, VerificationInput } from './types';
+
+export class PostDeploymentContractVerifier extends MultiGeneric<VerificationInput> {
```

Comment:
All verifiers are post deployment right? 😉

The name is probably fine, just pointing that out.